### ISSUE TITLE
Sound: add tts struct documentation

### ISSUE THREAD

**Issue by parkerbroz** [uchicago-cs/chiventure#1650]
After we have created the TTS struct, we want to provide documentation explaining its functionality. We will edit the README.md in src/sound to provide this documentation. This documentation will give an overview, covers basic structure, shows how to do an example demo, and includes info on the current status and future changes

Assignees: Parker, Rhea, Jake

**Comment by jwschroeder21** [comment]
Update: after receiving help from Borja in relation to Espeak and its not actually being required, we fixed the compiler errors we were dealing with. We have written all the documentation and are awaiting approval on our PR

**Comment by sangerr** [comment]
Closing Statement: In this issue, we provided documentation explaining the functionality of the tts structs. We edited the README.md in src/sound to provide this documentation. This documentation gave an overview, covered basic structure, showed how to do an example demo, and included info on the current status and future changes. We hope that this will help future teams in understanding how to integrate the text to speech features.
